app/services/event_log/move_runner.rb

```diff
@@ -16,7 +16,6 @@ def call
         when Event::APPROVE
           move.status = Move::MOVE_STATUS_REQUESTED
           move.date = event.date
-          move.determine_supplier
```

Comment:
Not sure I understand why we need to stop determining the supplier again if the effective date comparison might yield a different result in the supplier chooser. @smoothcontract I don't suppose you know why this has changed, do you?

Reply:
See https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/905 - explained there

Reply:
Okay, not sure what happens when you've both removed stuff.

Reply:
We'll get yours merged and then we can resolve conflicts